static/app/views/profiling/content.tsx

```diff
@@ -1,7 +1,56 @@
-type Props = {};
+import styled from '@emotion/styled';
+import {Location} from 'history';
 
-function ProfilingContent(_props: Props) {
-  return <h1>Profiling</h1>;
+import * as Layout from 'sentry/components/layouts/thirds';
+import NoProjectMessage from 'sentry/components/noProjectMessage';
+import PageFiltersContainer from 'sentry/components/organizations/pageFilters/container';
+import {normalizeDateTimeParams} from 'sentry/components/organizations/pageFilters/parse';
+import PageHeading from 'sentry/components/pageHeading';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import {t} from 'sentry/locale';
+import {PageContent} from 'sentry/styles/organization';
+import useOrganization from 'sentry/utils/useOrganization';
+
+import {ProfileScatterChart} from './landing/chart';
+import {ProfileTable} from './landing/table';
+
+interface Props {
+  location: Location;
 }
 
+function ProfilingContent({location}: Props) {
+  const organization = useOrganization();
+  const dateSelection = normalizeDateTimeParams(location.query);
+
+  return (
+    <SentryDocumentTitle title={t('Profiling')} orgSlug={organization.slug}>
+      <PageFiltersContainer>
+        <NoProjectMessage organization={organization}>
+          <StyledPageContent>
+            <Layout.Header>
+              <Layout.HeaderContent>
+                <StyledHeading>{t('Profiling')}</StyledHeading>
+              </Layout.HeaderContent>
+            </Layout.Header>
+            <Layout.Body>
+              <Layout.Main fullWidth>
+                <ProfileScatterChart {...dateSelection} traces={[]} />
+                <ProfileTable location={location} traces={[]} />
```

Comment:
I like how you lifted traces to the common props here 👍🏼  We'll probably need to add loading/error prop here as well, but we can easily do that later :)